src/foundry/prosemirror/schema/schema-definition.d.mts

```diff
@@ -0,0 +1,36 @@
+import { MarkSpec, Node, NodeSpec } from "prosemirror-model";
+
+export default SchemaDefinition;
+
+/**
+ * An abstract interface for a ProseMirror schema definition
+ */
+declare abstract class SchemaDefinition {
+  /**
+   * The HTML tag selector this node is associated with
+   */
+  static tag: string;
+
+  /**
+   * Schema attributes
+   */
+  static get attrs(): Record<string, any>;
```

Comment:
I believe these methods should all be individually tagged abstract as well (the ones that throw errors and have the typedoc `@abstract` tag)

Reply:
Yeah we do this because TypeScript doesn't allow the static side of a class to be abstract but in this case that's Foundry's intent.